api/controller.py

```diff
@@ -1958,6 +1959,8 @@ class ProfileController(CirculationManagerController):
     def _controller(self):
         """Instantiate a CoreProfileController that actually does the work.
         """
+        # TODO: Probably better to use request_patron and check for
+        # None here.
```

Comment:
It's not a good idea to call `authenticated_patron_from_request` and assume it will return a `Patron`, but that's what's happening here. I don't want to spend time fixing it in this branch though.